guidelines/index.html

```diff
@@ -913,9 +913,9 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
      <li>テキストは、支援技術なしで 200％までサイズ変更でき、利用者が<a href="#dfn-on-a-full-screen-window" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-on-a-full-screen-window-1" title="最も普及したサイズのデスクトップやラップトップのディスプレイで、ビューポートを最大化した状態。">全画面表示</a>にしたウィンドウで 1 行のテキストを読むときに横スクロールする必要がない。</li>   
   </ul>
 
-<div class="note" role="note" id="issue-container-generatedID-5"><div role="heading" class="note-title marker" id="h-note-5" aria-level="5"><span>注記 1</span></div><p class="">Content is not required to use these values. The requirement is that a mechanism is available for users to change these presentation aspects. The mechanism can be provided by the browser or other user agent. Content is not required to provide the mechanism.</p></div>
+<div class="note" role="note" id="issue-container-generatedID-5"><div role="heading" class="note-title marker" id="h-note-5" aria-level="5"><span>注記 1</span></div><p class="">コンテンツがこれらの値を用いる必要はない。要求事項は、利用者がこれらの提示の外観を変更するメカニズムを利用できることである。このメカニズムは、ブラウザ又は他のユーザエージェントによって提供されることがある。コンテンツがメカニズムを提供する必要はない。</p></div>
 
-<div class="note" role="note" id="issue-container-generatedID-6"><div role="heading" class="note-title marker" id="h-note-6" aria-level="5"><span>注記 2</span></div><p class="">Writing systems for some languages use different presentation aspects to improve readability and legibility. If a presentation aspect in this success criterion is not used in a writing system, content in that writing system does not need to use that presentation setting and can conform without it. Authors are encouraged to follow guidance for improving readability and legibility of text in their writing system.</p></div>
+<div class="note" role="note" id="issue-container-generatedID-6"><div role="heading" class="note-title marker" id="h-note-6" aria-level="5"><span>注記 2</span></div><p class="">言語の書記体系の中には、可読性及び視認性を向上させるためにさまざまな提示の外観を用いるものもある。ある書記体系でこの達成基準における提示の外観が用いられていない場合、その書記体系のコンテンツは、その提示の設定を用いる必要はなく、その提示の設定なしでも適合できる。コンテンツ制作者は、可読性及び視認性を向上させるために、言語の書記体系のガイドラインに従うことが推奨される。</p></div>
```

Comment:
「言語の書記体系の中には」とありますが、some は Writing systems ではなく languages にかかっているので、「言語の中には、書記体系で……」でしょうかね。

aspect は https://github.com/waic/wcag22/pull/46#discussion_r1423644560 で議論されていますが、これ外観なのかなぁ、という疑問は持ちました。"a presentation aspect in this success criterion" というのは、明らかにこの達成基準の5項目の箇条書き項目 (のうちのどれか1つ) を指しています。たとえば、箇条書き項目の3つ目は "Text is not justified" ですが、言語によっては justify という概念が存在しないこともありえて、その場合にはこの項目は見なくて良い、という話です。"presentation aspect" は、「この箇条書き項目で示されているような、presentationに関する観点」というようなことのはずです。

ということで「観点」に近いとは思うものの、これを「提示の観点」とするとわかりにくいのも確かであり、悩ましいですね……。
「概念」にしてみるとわかりやすい気がしましたがどうですかね? ご意見をお聞きしたいです。

```suggestion
<div class="note" role="note" id="issue-container-generatedID-6"><div role="heading" class="note-title marker" id="h-note-6" aria-level="5"><span>注記 2</span></div><p class="">言語の中には、書記体系において、可読性及び視認性を向上させるためにさまざまな提示の概念を用いるものもある。ある書記体系でこの達成基準における提示の概念が用いられていない場合、その書記体系のコンテンツは、その提示の設定を用いる必要はなく、その提示の設定なしでも適合できる。コンテンツ制作者は、可読性及び視認性を向上させるために、言語の書記体系のガイドラインに従うことが推奨される。</p></div>
```

Reply:
「外観」あまりしっくりきていませんでしたが、「概念」という言葉が当てられることでニュアンスが明瞭に伝わるようになった気がします。自分は賛成です！